Basic-Car-Maintenance/Shared/MainView/ViewModels/MainTabViewModel.swift

```diff
@@ -50,6 +47,11 @@ class MainTabViewModel {
                     self.alert = newAlert
                 }
             }
+
+        } catch {
+          print("Error getting documents: \(error)")
         }
+        
+
```

Comment:
don't add these extra lines, also you should've seen a SwiftLint warning for this! let me know if you didn't see this